onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -263,6 +275,10 @@ void RegisterConvActivationFusionRules(SelectorActionRegistry& registry) {
   auto selector = std::make_unique<selectors::ConvActivationSelector>();
   registry.RegisterSelectorAndAction(name, {{"Conv", {1, 11}}},
                                      std::move(selector), std::move(action));
+  auto selector2 = std::make_unique<selectors::ConvActivationSelector>();
+  auto action2 = std::make_unique<actions::FuseConvActivationAction>(kMSInternalNHWCDomain);
+  registry.RegisterSelectorAndAction("MSInternalNHWCConvAct", {{"Conv", {1, 11}}},
```

Comment:
we want to register the same names across minimal and full builds. otherwise, the minimal build would be unable to apply the action associated with that name.

the pattern should be:
```c++
#if !defined(ORT_MINIMAL_BUILD)
  registry.RegisterSelectorAndAction(name, ...);
#else
  registry.RegisterAction(name, ...);
#endif
```